Leanplum/Classes/LPStartResponse.m

```diff
@@ -26,32 +26,34 @@ - (NSDictionary *) classMapping {
 
 - (id) initWithDictionary:(NSDictionary *)responseDict {
     NSDictionary *mapping = [self classMapping];
-    for (id key in (responseDict[@"response"] ? responseDict[@"response"] : responseDict)) {
+    for (id key in responseDict) {
         if (mapping[key]) {
             if ([key isEqualToString: @"regions"]) {
-                NSMutableArray *regionArr = [[NSMutableArray alloc] init];
-                for (id regionDict in responseDict[mapping[key]]) {
```

Comment:
Actually when I wrote this I assumed the region response would be an array of regions, but currently backend might not return an array, if backend doesn't make the change this would need to be changed